terraform/environments/preview/main.tf

```diff
@@ -1,5 +1,6 @@
 provider "aws" {
   region = "eu-west-1"
+  version = "1.6.0"
```

Comment:
Will we need to add this version to the staging/production terraform files?

Reply:
ooo yes, good shout!